### PR TITLE
set base kvm attributes to point to right packages

### PR DESCRIPTION
**Pull request by knightsamar**

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -53,3 +53,5 @@
   only_if { platform_family?('rhel') }
   action :create
 end
+
+node.default['base']['kvm']['qemu-pkgs'] = ['qemu-kvm-ev', 'qemu-kvm-tools-ev']
```
Comment (ramereth) [inline recipes/default.rb]: Mind changing this to use ``%w()`` syntax instead?

**Review by ramereth — CHANGES_REQUESTED** [review]

**Review by ramereth — APPROVED** [review]
LGTM, please squash commits and then I'll merge it.

**Comment by knightsamar** [comment]
@ramereth Done!

**Comment by ramereth** [comment]
!bump patch *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v1.1.3.  Have a nice day!

**Pull request merged**